src/workerd/api/pyodide/pyodide.c++

```diff
@@ -25,7 +27,11 @@ void PyodideBundleManager::setPyodideBundleData(
     kj::String version, kj::Array<unsigned char> data) const {
   auto wordArray = kj::arrayPtr(
       reinterpret_cast<const capnp::word*>(data.begin()), data.size() / sizeof(capnp::word));
-  auto messageReader = kj::heap<capnp::FlatArrayMessageReader>(wordArray).attach(kj::mv(data));
+  // We're going to reuse this in the ModuleRegistry for every Python isolate, so set the traversal
```

Comment:
Note this is only safe if this data is trusted. I assume it is since it's shared python data, not from a user...